### PR TITLE
Add a simple binary Dart script

### PR DESCRIPTION
**Pull request by srawlins**
I use this library often enough that I'd love to just be able to run it from the command line.

This is a crazy simple script that either reads an argument as a file, or reads from stdin, and spits HTML out to stdout.

I have no intention to add more features to this, but there will probably be requests :stuck_out_tongue_closed_eyes:

### CODE REVIEW

File: bin/markdown.dart [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import 'dart:io';
+
+import 'package:markdown/markdown.dart';
+
+void main(List args) async {
```
Comment (munificent) [inline bin/markdown.dart]: `List<String>`

File: bin/markdown.dart [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  if (args.length == 1) {
+    // Read argument as a file path.
+    print(markdownToHtml(new File(args[0]).readAsStringSync()));
+    exit(0);
+  }
+
+  // Read from stdin.
+  StringBuffer s = new StringBuffer();
+  String line;
```
Comment (munificent) [inline bin/markdown.dart]: Use `var` for these, and how about `buffer` instead of just `s`?

**Comment by srawlins** [comment]
Done. Thanks for the review!

**Comment by munificent** [comment]
:+1: LGTM!

**Pull request merged**